mod/build.gradle

```diff
@@ -405,7 +418,7 @@ task copyReleaseBuilds(dependsOn: buildReleaseNatives) {
 processResources.mustRunAfter copyReleaseBuilds
 build.dependsOn copyReleaseBuilds
 
-task buildDevLibrary() {
+task buildDevLibrary(dependsOn: updateSubmodules) {
```

Comment:
Will this try to update the submodules every time? How will gradle know that the submodules are already downloaded?

Reply:
git knows so its fine

Reply:
(I tested)